examples/tutorial_client_eventfilter.c

```diff
@@ -0,0 +1,267 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+/**
+ * In this C file we will create a ClientFilter example.
+ * Client configured on the server
```

Comment:
Please add a newline between licence and description. The description can be more precise: In this tutorial a client setup an event filter on the server .. You should also mention the server event filter example here.

Reply:
We have updated the description.